Automattic/wp-calypso
# Move reader-main to reader/components

### Changes proposed in this Pull Request
A number of Reader-specific components and blocks currently live in the main components and blocks directories, even though they are not used elsewhere in Calypso. I feel that these components and blocks should live inside client/reader if they are not likely to be reused outside Reader. See also: #28161.

### What's reader-main anyway?

> This class is used by pieces of the Reader to indicate they want "editorial" styles. Notably, this overrides the background color of the document and is used as a hook by other parts to override styles.

### Testing instructions
Load the Reader at http://calypso.localhost:3000 and ensure that all Reader streams have a white background.
